### PR TITLE
add cell renderer for links

### PR DESCRIPTION
**Pull request by Lendemor**

### CODE REVIEW

File: custom_components/reflex_ag_grid/ag_grid.py [hunk 1]
```diff
@@ -67,6 +67,12 @@ class AGEditors(SimpleNamespace):
     checkbox = "agCheckboxCellEditor"
 
 
+class AGRenderers(SimpleNamespace):
+    link = rx.Var(
+        """function(params) {return jsx("a", {href: params.value}, params.value);}"""
```
Comment (masenf) [inline custom_components/reflex_ag_grid/ag_grid.py:72]: need `VarData`

```suggestion
        """function(params) {return jsx("a", {href: params.value}, params.value);}""",
        var_data=VarData(
            imports={
                "@emotion/react": [
                    ImportVar(tag="jsx"),
                ],
            }
        ),
```

**Review by masenf — COMMENTED** [review]
the thing i dont _love_ about this is there's no way to set the link title, but this is good enough for the short term and solves the problem. we can get it into today's release and see if users have additional feedback.

**Comment by masenf** [comment]
@Lendemor should we open the link in a new window?

File: custom_components/reflex_ag_grid/ag_grid.py [hunk 2]
```diff
@@ -67,6 +67,19 @@ class AGEditors(SimpleNamespace):
...
+    link = rx.Var(
+        """function(params) {return jsx("a", {href: params.value, target: "blank"}, params.value);}""",
+        _var_data=rx.vars.VarData(
+            imports={
+                "@emotion/react": [
+                    rx.ImportVar(tag="jsx"),
+                ],
+            }
+        ),
+    )
```
Comment (adhami3310) [inline custom_components/reflex_ag_grid/ag_grid.py:80]:
```suggestion
    params = rx.Var("params").to(dict)
    link = rx.vars.function.ArgsFunctionOperation.create(
        ("params",),
        rx.link(params.value, href=params.value, target="_blank")
        ),
    )
```

**Review by masenf — APPROVED** [review]

**Pull request merged**